examples/client.php

```diff
@@ -111,6 +118,27 @@ public function createTransactionExample() {
         }
     }
 
+    public function getTransactionByExternalIdExample() {
+        $transactionsApi = new TransactionsApi();
+        $externalId = 'TRANSACTION-1f834adf';
+        try {
+            $transactionResponse = $transactionsApi->getTransactions(null, null, $externalId);
+            $transaction = reset($transactionResponse->getObject());
+            echo "Transaction found! External ID: {$transaction->getExternalID()}", PHP_EOL;
+            return $transaction->getExternalId();
```

Comment:
The transaction should be returned here (as per the method name) rather than the external ID